### PR TITLE
LRQA-81142 - Automation CanSelectItemActionsAsFieldToExportWithJSONT

### PR DESCRIPTION
**Pull request by pedroamarall**
cc: @manoelcyreno

### CODE REVIEW

**Comment by pedroamarall** [comment]
ci:test:sf

**Comment by pedroamarall** [comment]
ci:help

**Review by magjed4289 — CHANGES_REQUESTED** [review]

File: modules/apps/batch-planner/batch-planner-test/src/testFunctional/paths/ImportExport.path [hunk 1]
```diff
@@ -20,6 +20,11 @@
 	<td>//table[contains(@class,'table-list')]//td[contains(@class,'lfr-name-column')]/a[contains(text(),'${key_executionName}')]/following::td[contains(@class,'${key_column}-column')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>ASSERT_FIELDS</td>
```
[the remark this answers is not in the document]
Comment (pedroamarall) [reply modules/apps/batch-planner/batch-planner-test/src/testFunctional/paths/ImportExport.path]: Hi @magjed4289 , this element is not a button but anyway I changed it to this name which reflects more the functionality of the path, thanks!

**Comment by pedroamarall** [comment]
ci:test:sf

**Review by magjed4289 — APPROVED** [review]

**Comment by magjed4289** [comment]
PR approved! Thanks @pedroamarall! 👏🏻

**Review by magjed4289 — APPROVED** [review]

**Comment by magjed4289** [comment]
ci:forward

**Pull request closed without merging**